Sources/PointFree/Mailgun/Mailgun.swift

```diff
@@ -0,0 +1,96 @@
+import Either
+import Foundation
+import HttpPipeline
+import Optics
+import Prelude
+
+enum Tracking: String {
```

Comment:
the contents of this file could be extracted to a lil swift mailgun package.

I lifted some mailgun parameters into proper types and made some functions for hitting the api